pkg/kubelet/kubelet.go

```diff
@@ -203,6 +203,9 @@ const (
 
 	// instrumentationScope is the name of OpenTelemetry instrumentation scope
 	instrumentationScope = "k8s.io/kubernetes/pkg/kubelet"
+
+	// urgentPodKillTimeoutSeconds is how long an urgent pod kill should wait for the pod to be killed before timing out
+	urgentPodKillTimeoutSeconds = 300
```

Comment:
Do we want to make this configurable?

Reply:
`urgentPodKillTimeoutSeconds` makes me think of some kind of urgency around it. However, as far as I understood, this will be applied for all pods. May be its just me. Let's see what other reviewers will say.

Reply:
With "urgent" I was hoping to convey that there is some time sensitivity to waiting on the pod, because the node is under pressure and we want to wait, but not wait too long